src/core/common/trickle_timer.hpp

```diff
@@ -109,6 +109,36 @@ class TrickleTimer : public TimerMilli
      */
     Mode GetMode(void) const { return mMode; }
 
+    /**
+     * This method gets the current interval min value of the trickle timer.
+     *
+     * @returns The current interval min value of the trickle timer.
+     */
+    uint32_t GetIntervalMin(void) const { return mIntervalMin; }
+
+    /**
+     * This method gets the current interval max value of the trickle timer.
+     *
+     * @returns The current interval max value of the trickle timer.
+     */
+    uint32_t GetIntervalMax(void) const { return mIntervalMax; }
```

Comment:
nit: is this used somewhere?

Reply:
No. I was using this for some logging during my dev work, but it's not needed. I'll remove it.